devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -40,5 +40,26 @@
       owner: vagrant
       group: vagrant
 
+- name: Install dummy data deps
+  pip:
+    name:
+      - python-freeipa
+      - Faker
 
+- name: Create dummy freeipa data
+  shell: python3 /vagrant/devel/ansible/roles/dev/create-test-data.py
```

Comment:
I think this could have been directly in `/devel` so it's more easily accessible in vagrant itself, and because it has nothing to do with ansible (it's not copied through ansible).